_data/meltano/utilities/cratedb/cratedb.yml

```
@@ -0,0 +1,42 @@
+capabilities: []
+commands:
+  start:
+    args: start
+    container_spec:
+      image: crate:5.5
```

Probably use `latest` here?

```suggestion
      image: crate:latest
```

---

Switched to use `crate:latest` now.